github/VisualStudio
# Explicitly open repository when clone completes

When a clone completes in Visual Studio 2017, the repository context would only consistently change if there was no solution or folder open. This PR explicitly opens the repository when the clone completes. This makes the behavior consistent with Visual Studio 2015.

- Explicitly open repository when clone completes in Visual Studio 2017
- Use `ITeamExplorerServices.OpenRepository` for both Visual Studio 2015 and 2017 

### How to repro

1. Open a solution in Visual Studio 2017
2. Open the `Clone` dialog
3. Start cloning a repository
4. Expect `Team Explorer` and clone progress bar to appear
5. 🐛 The active repository never changes in `Team Explorer`

### How to test the fix

1. Open a solution in Visual Studio 2017
2. Open the `Clone` dialog
3. Start cloning a repository
4. Expect `Team Explorer` and clone progress bar to appear
5. If there were changes in active solution, expect dialog with option to save
6. Expect cloned repository to appear in Solution Explorer folder view
7. Expect new repository to appear on `Team Explorer` (with list of solutions to open)

Fixes #2003
